src/test/server/ServerStatus_test.cpp

```diff
@@ -508,6 +510,110 @@ class ServerStatus_test :
         }
     }
 
+    void
+    testAmendmentBlock(boost::asio::yield_context& yield)
+    {
+        testcase("Status request over WS and RPC with/without Amendment Block");
+        using namespace jtx;
+        using namespace boost::asio;
+        using namespace beast::http;
+        Env env {*this, validator( envconfig([](std::unique_ptr<Config> cfg)
+            {
+                cfg->section("port_rpc").set("protocol", "http,https");
+                return cfg;
+            }), "")};
+
+        env.close();
+
+        // advance the ledger so that server status
+        // sees a published ledger -- without this, we get a status
+        // failure message about no published ledgers
+        env.app().getLedgerMaster().tryAdvance();
+
+        // make an RPC server info request and look for
+        // amendment_blocked status
+        auto si = env.rpc("server_info") [jss::result];
+        BEAST_EXPECT(! si[jss::info].isMember(jss::amendment_blocked));
+        BEAST_EXPECT(
+            env.app().getOPs().getConsensusInfo()["validating"] == true);
+
+        auto const port_ws = env.app().config()["port_ws"].
+            get<std::uint16_t>("port");
+        auto const ip_ws = env.app().config()["port_ws"].
+            get<std::string>("ip");
+
+
+        boost::system::error_code ec;
+        response<string_body> resp;
+
+        doRequest(
+            yield,
+            makeHTTPRequest(*ip_ws, *port_ws, ""),
+            *ip_ws,
+            *port_ws,
+            false,
+            resp,
+            ec);
+
+        if(! BEAST_EXPECTS(! ec, ec.message()))
+            return;
+        BEAST_EXPECT(resp.status == 200);
+        BEAST_EXPECT(
+            resp.body.find("connectivity is working.") != std::string::npos);
+
+        // mark the Network as Amendment Blocked, but still won't fail until
+        // ELB is enabled (next step)
```

Comment:
I had trouble finding where `ELB_SUPPORT` is documented.   Based on this test, it seems to disable RPC calls once the server is blocked?